docs/pages/installation/amazon-ecs.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Installing Teleport on Amazon ECS using Terraform
+description: How to install and run Teleport agents on Amazon ECS
+labels:
+ - platform-wide
+ - reference
+---
+
+Using Amazon ECS and Terraform allows you to easily deploy and manage Teleport Agents that are closer to your AWS resources.
+
+This flow is ideal when you want to auto discover and secure access to your EKS clusters, RDS databases or other supported AWS resources.
+
+## How it works
+Amazon ECS provides a convenient way to deploy Teleport agents closely located to your AWS resources, allowing access even if they are not publicly accessible.
+With Terraform, you can easily deploy and keep the Teleport agent up to date.
```

Comment:
For me, this section rephrases the introductory paragraph. Instead, I would make it clear to an admin what the architecture is of the setup introduced in this guide so they understand, before following the steps, what technical things they'll be asked to do.

In particular, what infrastructure does the Terraform module deploy? I'm guessing an ECS service for a Teleport Agent? Does the module also create a join token resource for the agent? Is there anything else?